src/commands/deployWorkspaceProject/getDefaultValues/getDefaultContainerAppsResources.ts

```diff
@@ -56,3 +67,38 @@ export async function getDefaultContainerAppsResources(context: ISubscriptionAct
         return noMatchingResources;
     }
 }
+
+async function promptForContainerAppsEnvironmentResources(context: ISubscriptionActionContext): Promise<DefaultContainerAppsResources> {
+    const client: ContainerAppsAPIClient = await createContainerAppsAPIClient(context)
+    const managedEnvironments: ManagedEnvironment[] = await uiUtils.listAllIterator(client.managedEnvironments.listBySubscription());
+
+    if (!managedEnvironments.length) {
+        return noMatchingResources;
```

Comment:
If there are no managed environments, shouldn't we still prompt with only `$(plus) Create new container apps environment` as the option?

Reply:
Nah, just a reminder since it's probably been a while since the naming PR, but an `undefined` resource means that we were unable to acquire an existing resource to use.  Therefore, later, the undefined resource will indicate to the command that we need to create the missing resource, thus in this scenario, returning `undefined` is equivalent to the user selecting the create button.  Since there's only one option, we can just pick it for them and skip displaying the prompt.

Reply:
I think I get how this is supposed to work now. I honestly really don't like this design pattern-- I find it's super confusing and not very congruent with how we would normally do it with wizard steps.

That being said, I understand that we've been kicking this can on the release for quite a while, so let's just get this in. We can always refactor into wizard steps later.

Reply:
Yeah, given the time restraints and the potential for side effects, I think it's just easier for now to go with the implementation like this.  After release we can add an engineering task to refactor this into a list step and throw it into our normal wizard flow.